src/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  height: 403px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card_image {
+  content: url("./images/imac.jpeg");
+  width: 160px;
+  height: 134px;
+  margin: 30px 0;
+}
+
+.product-card_content-block {
+  padding: 0 15px;
+}
+
+.product-card_description-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-top: 15px;
+  margin-bottom: 6px;
+}
+
+.product-card_product-code {
+  width: 100%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 12px;
+  margin-bottom: 16px;
+}
+
+.product-card_review-block {
+  display: flex;
+}
+
+.review-block_stars {
+  display: flex;
+  justify-content: space-around;
+  width: 91px;
+  height: 15px;
+  margin-top: 1px;
+}
+
+.review-block_stars_star_item {
+  width: 13px;
+  height: 12px;
+  content: url("./images/star.svg");
+}
+
+.review-block_stars :nth-child(-n + 4) {

```

Comment:
add rules to -n+1, -n+2 and so on, so when I change class - count of active stars also changes